code/modules/mod/mod_control.dm

```diff
@@ -135,6 +135,8 @@
 		piece.permeability_coefficient = theme.permeability_coefficient
 		piece.siemens_coefficient = theme.siemens_coefficient
 		piece.icon_state = "[skin]-[initial(piece.icon_state)]"
+		piece.icon = theme.icon_override
+		piece.worn_icon = theme.worn_icon_override
```

Comment:
this is run only on initialize, put it on set_mod_skin too

Reply:
Resolved